lib/src/widgets/chat/ds_document_select.widget.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import '../../models/ds_document_select.model.dart';
+import '../../utils/ds_message_content_type.util.dart';
+
+class DSDocumentSelect extends StatelessWidget {
```

Comment:
Would it be worth documenting all widgets and their attributes?

Reply:
done